model-engine/model_engine_server/db/migrations/alembic/env.py

```diff
@@ -0,0 +1,99 @@
+import logging
+import os
+from logging.config import fileConfig
+
+from alembic import context
+from model_engine_server.db.base import get_engine_url
+from sqlalchemy import engine_from_config, pool
+
+env = os.environ.get("ENV")
+assert env is not None, "Expected ENV to be a nonempty environment variable."
+
+# this is the Alembic Config object, which provides
+# access to the values within the .ini file in use.
+config = context.config
+
+config.set_main_option("sqlalchemy.url", get_engine_url(env, read_only=False).url)
+
+ALEMBIC_TABLE_NAME = "alembic_version_model_engine"
+
+# Interpret the config file for Python logging.
+# This line sets up loggers basically.
+if config.config_file_name is not None:
+    fileConfig(config.config_file_name)
+
+# add your model's MetaData object here
+# for 'autogenerate' support
+# from myapp import mymodel
+# target_metadata = mymodel.Base.metadata
+target_metadata = None
```

Comment:
is it possible for us to use existing ORM objects for autogen?

Reply:
yes; just need to set it up